### PR TITLE
[Merged by Bors] - Fix: node selector

### PR DESCRIPTION
**Pull request by fhennig**
# Description
Parent ticket: https://github.com/stackabletech/issues/issues/300

## Review Checklist

- [x] Code contains useful comments
- ~~CRD change approved~~ (not applicable, no changes)
- ~~(Integration-)Test cases added~~ (not applicable, see parent ticket)
- ~~Documentation added~~ (not applicable: bugfix)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/examples/code/trino.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   name: simple-trino
 spec:
   image:
-    productVersion: 396
+    productVersion: "396"
```
Comment (maltesander) [inline docs/modules/getting_started/examples/code/trino.yaml:8]: We could go for the "latest" version which is "403-stackable0.1.0"

**Review by maltesander — APPROVED** [review]
LGTM.

File: docs/modules/getting_started/examples/code/trino.yaml (hunk 1 above)
Comment (fhennig) [reply docs/modules/getting_started/examples/code/trino.yaml:8]: not here

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**